src/main/java/com/PhamKornbluhGroup/APIResultData.java

```diff
@@ -0,0 +1,67 @@
+package com.PhamKornbluhGroup;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.nio.charset.StandardCharsets;
+
+public final class APIResultData {
+    private int responseCode;
+    private String responseMessage;
+    private String encoding;
+    private String content;
+
+    public APIResultData() {
+    }
+
+    public void initResultData(HttpURLConnection connection) {
+        StringBuilder builder = new StringBuilder();
+        //b responseStream autocloses
+        //b reader autocloses
+        try (InputStream responseStream = (InputStream) connection.getContent(); // TODO: see if better --> connection.getInputStream()
+             BufferedReader reader = new BufferedReader(new InputStreamReader(responseStream, StandardCharsets.UTF_8));) {
```

Comment:
Try removing the semicolon to see if the code is still syntactically correct.

Reply:
It is still syntactically correct without the extra ;

Removed. Thanks!